packages/tasks/src/model-libraries.ts

```diff
@@ -372,6 +372,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: {
 			wildcard: { path: "*.sentis" },
 		},
+	"whisperkit": {
```

Comment:
This broke the filter, it's missing a closing one

```suggestion
	},
	"whisperkit": {
```